package.json

```diff
@@ -168,7 +167,8 @@
       "decoderWorker\\.min\\.js": "<rootDir>/node_modules/matrix-react-sdk/__mocks__/empty.js",
       "decoderWorker\\.min\\.wasm": "<rootDir>/node_modules/matrix-react-sdk/__mocks__/empty.js",
       "waveWorker\\.min\\.js": "<rootDir>/node_modules/matrix-react-sdk/__mocks__/empty.js",
-      "context-filter-polyfill": "<rootDir>/node_modules/matrix-react-sdk/__mocks__/empty.js"
+      "context-filter-polyfill": "<rootDir>/node_modules/matrix-react-sdk/__mocks__/empty.js",
+      "FontManager.ts": "<rootDir>/node_modules/matrix-react-sdk/__mocks__/FontManager.js"
```

Comment:
.ts shirley?

Reply:
or hsa it been compiled down to .js by this point?

Reply:
oh, no the mock really is .js, even though it's mocking a .ts file. nm.

Reply:
Yeah whilst `ts` probably would have worked, files outside `src` don't count on our graphs anyway. I fear Jest would need some coercing to be happy with TS there though did not bother trying (rest of mocks are JS too)